.tekton/kueue-bundle-main-pull-request.yaml

```diff
@@ -82,7 +82,7 @@ spec:
       description: Execute the build with network isolation
       name: hermetic
       type: string
-    - default: ""
+    - default: '{"type": "rpm", "path": "./konflux"}'
```

Comment:
Why change only the bundle pull but change the rest of the push files for the other components?